scripts/enable-forker-travis.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Setup TravisCi for forker (contributor):
+# The setup will allow forkers to automatically deploy
+# their WIP branches to npm.
+#
+# Useful when you have a WIP PR in foreman or in a foreman plugin.
+
+set -e
+
+REPO_NAME="foreman-js"
+
+echo 'Setting up travis for forker account, please follow instructions below.'
+echo 'Please enter your github username:'
+read GH_USERNAME
+echo -e '\n'
+
+echo 'Please create a token for your github account.'
+echo 'See: https://github.com/settings/tokens'
+echo 'Your token should contain the following scopes:'
+echo 'public_repo, user:email, repo:status, admin:repo_hook'
+echo 'Your github token:'
+read GH_TOKEN
+echo -e '\n'
+
+echo 'Please create a token for your npm account'
+echo 'See: https://docs.npmjs.com/creating-and-viewing-authentication-tokens'
+echo 'Your npm token:'
+read NPM_TOKEN
+echo -e '\n'
+
+REPO_SLUG="${GH_USERNAME}/${REPO_NAME}"
+
+gem install travis --quiet
+travis login --github-token $GH_TOKEN
```

Comment:
This will fail if you don't have a fork of the `foreman-js` repo on your github.  (I had checked it out from your PR in DETACHED HEAD state but I hadn't forked it yet.)  May want to add a reminder here that you must have forked before continuing.